azurerm/internal/services/bot/parse/bot_channels_registration.go

```diff
@@ -0,0 +1,33 @@
+package parse
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type BotChannelsRegistrationId struct {
+	ResourceGroup string
+	Name          string
```

Comment:
(as above)

Reply:
Confused on this one? The `Name` field is already exposed. The other channels are connected to this one, so I shouldn't need `BotName` correct? 🤔